### PR TITLE
Update styles for dropdown

### PR DESCRIPTION
**Pull request by jdetle**
This should cause everything to look nice for the CSS themes.
css_fixes

### CODE REVIEW

**Comment by rgbkrk** [comment]
I wonder when/how the inputs broke for classic mode.

File: static/styles/main.css [hunk 1]
```diff
@@ -435,7 +433,7 @@ li.CodeMirror-hint-active {
     padding-right: 0px;
     font-size: 11px;
     margin: 20px 0;
-    background-color: rgba(255,255,255,1);
+    background-color: var(--dropdown-content);
```
Comment (rgbkrk) [inline static/styles/main.css:435]: 👍

File: static/styles/theme-christmas.css [hunk 2]
```diff
@@ -12,6 +12,9 @@
   --toolbar-button: #aaa;
   --toolbar-button-hover: #555;
 
+  --dropdown-content: white;
+  --dropdown-content-hover: grey;
```
Comment (rgbkrk) [inline static/styles/theme-christmas.css]: Heh.

**Comment by jdetle** [comment]
¯_(ツ)_/¯

File: static/styles/theme-christmas.css (hunk 2 above)
Comment (jdetle) [reply static/styles/theme-christmas.css]: do you prefer "gray"?

Comment (jdetle) [reply static/styles/theme-christmas.css]: its a little heavy actually

Comment (rgbkrk) [reply static/styles/theme-christmas.css]: Nah, I'm giggling because you had to update our hidden themes. :)

Comment (captainsafia) [reply static/styles/theme-christmas.css]: I think the gray we originally used for this was #ecf0f1.

**Comment by rgbkrk** [comment]
I seriously have no idea why Travis is not behaving well today. Since these are just CSS changes that don't affect our CSS structure (which is specced), I'm going to go ahead and merge.

**Pull request merged**